cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,104 @@
-/// <reference types='cypress' />
+describe('Web Tables Page', () => {
+  const worker = {
+    firstName: 'Mykhailo',
+    lastName: 'Rozmaznin',
+    email: 'rozmaznin@gmail.com',
+    age: '19',
+    salary: '30000',
+    department: 'QA',
+  };
 
-describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should have the pagination', () => {
+    
+    cy.contains('Page').should('exist');
+    cy.get('select').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-totalPages').should('exist');
+    cy.get('.-pageJump > input').should('exist');
+    cy.get('.-pageInfo').should('exist');
+  });
+  
+
+  it('should contain rows', () => {
```

Comment:
what does this test check? An idea was, that you select 5 rows in pagination and test that only 5 rows are displayed